integration-tests/testconfig/common/vrf/common.go

```diff
@@ -0,0 +1,297 @@
+package vrf
+
+import (
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/pkg/errors"
+
+	"github.com/smartcontractkit/chainlink-testing-framework/blockchain"
+)
+
+type Config struct {
+	Common            *Common            `toml:"Common"`
+	General           *General           `toml:"General"`
+	ExistingEnvConfig *ExistingEnvConfig `toml:"ExistingEnv"`
+	Performance       *PerformanceConfig `toml:"Performance"`
+}
+
+const (
+	ErrDeviationShouldBeLessThanOriginal = "`RandomnessRequestCountPerRequestDeviation` should be less than `RandomnessRequestCountPerRequest`"
+)
+
+func (c *Config) Validate() error {
+	if c.Common != nil {
+		if err := c.Common.Validate(); err != nil {
+			return err
+		}
+	}
+	if c.General != nil {
+		if err := c.General.Validate(); err != nil {
+			return err
+		}
+	}
+	if c.Performance != nil {
+		if err := c.Performance.Validate(); err != nil {
+			return err
+		}
+	}
+
+	return nil
```

Comment:
we don't want to validate `ExistingEnvConfig`?

Reply:
good catch